common/mqtt_client/tests/packet_size_tests.rs

```diff
@@ -15,7 +15,7 @@ enum TestJoinError {
 async fn packet_size_within_limit() -> Result<(), anyhow::Error> {
     // Start the local broker
     let mqtt_server_handle = tokio::spawn(async {
-        start_broker_local("../../configuration/rumqttd/rumqttd_5883.conf").await
+        rumqttd_broker::start_broker_local("../../configuration/rumqttd/rumqttd_5883.conf").await
```

Comment:
I guess the result is ignored on purpose: this can fail because the port is already used by another instance of the test broker. I would make this clearer at least with a comment.

Reply:
The broker and the port are specific to each test. If two instances of the same test run, then I think then there should be a problem. I will put a comment there.

Reply:
Oh, a config file per test! This will not scale.